tests/graphql/taxNumber.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+
+import { Client } from "../../lib";
+import { TaxNumberType } from "../../lib/graphql/schema";
+import { TaxNumber } from "../../lib/graphql/taxNumber";
+
+const taxNumberData = {
+  id: "921be5e1-cefb-4b91-9633-fcadca94bd72",
+  type: TaxNumberType.Personal,
+  taxNumber: "2388081508158",
+  modificationDate: new Date("2022-01-01"),
+  description: "personal tax number",
+};
+
+describe("TaxNumber", () => {
+  let sandbox: sinon.SinonSandbox;
+  let client: Client;
+
+  beforeEach(async () => {
+    sandbox = sinon.createSandbox();
+
+    const clientId = "26990216-e340-4f54-b5a5-df9baacc0440";
+    const redirectUri = "https://localhost:3000/auth/callback";
+    const scopes = ["transactions"];
+    const state = "25843739712322056";
+
+    client = new Client({
+      clientId,
+      redirectUri,
+      scopes,
+      state,
+    });
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("#fetch", () => {
+    it("should call rawQuery and return all tax numbers details", async () => {
+      // arrange
+      const taxNumbersData = [
+        taxNumberData,
+        { ... taxNumberData, id: "b9ff9d3d-3ab7-452e-a16c-e1611fe443aa" },
```

Comment:
🙃 It looks strange when there is space after `...`
```suggestion
        { ...taxNumberData, id: "b9ff9d3d-3ab7-452e-a16c-e1611fe443aa" },
```
💭 We probably should enforce some linting rules